packages/web/pages/user/my-account/index.js

```diff
@@ -18,12 +19,22 @@ const MyProfile = () => {
 	const [passwordMessage, setPasswordMessage] = useState('');
 	const [passwordLoading, setPasswordLoading] = useState(false);
 
-	const handleSubmit = async (data) => {
+	const handleSubmit = async ({ cpf, zipcode, birth_date, ...data }) => {
 		setLoading(true);
-		const result = await updateUser(user.id, data);
+		const result = await updateUser(user.id, {
+			...data,
+			cpf: unMask(cpf),
+			zipcode: unMask(zipcode),
+			birth_date: stringToDate(birth_date),
+		});
 		setLoading(false);
-		if (!result) {
-			setMessage(t('account:messages.error'));
+
+		if (result.error) {
```

Comment:
![image](https://user-images.githubusercontent.com/6104632/87558622-b35c1200-c68f-11ea-843f-1833f5d82b24.png)

```suggestion
		if (result?.error) {
```

Reply:
O erro resolveu quando refiz o seed da api mas o codigo precisa estar mais resiliente. O front não pode quebrar se a API quebrar.

Reply:
verdade :+1:

Reply:
Geralmente coloco, mas foi falta de atenção mesmo :+1: